### PR TITLE
Update dependencies.

### PR DESCRIPTION
**Pull request by tjprescott**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,8 +40,8 @@
   },
   "types": "./dist/src/index.d.ts",
   "dependencies": {
-    "@azure-tools/typespec-azure-core": "0.43.0",
-    "@azure-tools/typespec-autorest": "0.43.0",
+    "@azure-tools/typespec-azure-core": ">=0.44.0, <1.0.0",
+    "@azure-tools/typespec-autorest": ">=0.44.0, <1.0.0",
```
Comment (catalinaperalta) [inline package.json:44]: Would breaks in either of these packages break or affect your tool in any way?

Comment (tjprescott) [reply package.json:44]: Not likely, no. And they always bump the minor version whether it's breaking or not, and I definitely don't want to have to do a token release just to bump the dependency.

There are no actual imports from either package in the code. We simply use autorest to generate the Swagger.

**Review by catalinaperalta — APPROVED** [review]
Not seeing the error handling in this PR but assuming it's already in main?

**Comment by tjprescott** [comment]
> Not seeing the error handling in this PR but assuming it's already in main?

That's right

**Pull request merged**